StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -14,14 +14,17 @@ import UIKit
 protocol EmbeddedPaymentMethodsViewDelegate: AnyObject {
     func heightDidChange()
     func selectionDidUpdate()
+    func presentSavedPaymentMethods(selectedSavedPaymentMethod: STPPaymentMethod?)
 }
 
 /// The view for an embedded payment element
 class EmbeddedPaymentMethodsView: UIView {
 
     typealias Selection = VerticalPaymentMethodListSelection // TODO(porter) Maybe define our own later
 
+    private var savedPaymentMethod: STPPaymentMethod?
```

Comment:
Per the dead code detection bot, this appears assigned but never used.

Reply:
🤔 -- indeed, we can get away with not keeping it as an instance variable